src/pal/inc/unixasmmacrosarm.inc

```diff
@@ -205,6 +192,31 @@ C_FUNC(\Name\()_End):
         .inst.w 0xde01
 .endm
 
+.macro PROLOG_PUSH RegList
+        push_nonvol_reg "\RegList"
+.endm
+
+.macro PROLOG_VPUSH RegList
+        vpush_nonvol_reg "\RegList"
+.endm
+
+.macro PROLOG_STACK_SAVE Register
+        .setfp \Register, sp
+        mov \Register, sp
+.endm
+
+.macro EPILOG_STACK_RESTORE Register
+        mov sp, \Register
```

Comment:
It seems we would also need an annotation here to let the unwinder know the sp was restored.